portlets/src/main/frontend/src/apps/components/rule/RuleList.vue

```diff
@@ -44,24 +123,27 @@
                                 <label class="switch">
                                     <input type="checkbox" v-model="rule.enabled">
                                     <span class="slider round"></span>
-                                    <span class="absolute-no">NO</span>
+                                    <span class="absolute-no">{{$t(`exoplatform.gamification.YES`)}}</span>
```

Comment:
replacing NO by YES ?

Reply:
hamadi requested  to change the default by yes